### PR TITLE
Feat/filters split and first

### PR DESCRIPTION
**Pull request by sassman**
fix for #224

summary:
- upgrades liquid to v0.20
- enables all liquid default filters
- fix some findings of clippy

### CODE REVIEW

**Comment by sassman** [comment]
To: @ashleygwilliams 
Cc: @k0pernicus 

After looking around I found that this PR fixes issue #212 and close also PR #213 as a side effect.

Can I help to get this thing merged?

**Comment by sassman** [comment]
Is there any chance that we can get that PR reviewed?

**Comment by k0pernicus** [comment]
Hi @sassman 
Very sorry for the delay...

I will try to take back this project as soon as possible, and make sure to review your PR this weekend :)

Thanks for your PR!

File: tests/integration/basics.rs [hunk 1]
```diff
@@ -803,11 +803,13 @@ fn it_applies_filters() {
             r#"kebab-case = {{crate_name | kebab_case}}
     PascalCase = {{crate_name | pascal_case}}
     snake_case = {{crate_name | snake_case}}
+    without_suffix = {{crate_name | split: "_" | first}}
     "#,
         )
         .init_git()
         .build();
     let dir = dir("main").build();
+    // without_suffix = {{crate_name | split "_project" | first}}
```
Comment (k0pernicus) [inline tests/integration/basics.rs:812]: Can you remove this comment please?

**Comment by k0pernicus** [comment]
Thanks @sassman, this PR is good for me :)
Thanks for updating liquid + updating doc and tests! 👏

**Comment by k0pernicus** [comment]
This PR will necessitates a new tag version soon I think... cc @ashleygwilliams

**Comment by sassman** [comment]
@k0pernicus thanks for merging. Was my pleasure to help.

**Comment by softprops** [comment]
+1 Thanks for merging!

**Pull request merged**